### PR TITLE
fix(android): support Geolocation altitudeAccuracy

### PR DESCRIPTION
**Pull request by garymathews**
- Include support for `altitudeAccuracy` on Android API 26+

##### TEST CASE
```JS
const win = Ti.UI.createWindow({ backgroundColor: 'gray' });
const listView = Ti.UI.createListView();
let count = 1;

Ti.Geolocation.accuracy = Ti.Geolocation.ACCURACY_HIGH;

function push (e) {
    let section = Ti.UI.createListSection({ headerTitle: '#' + count++ });
    if (e.success) {
        if (e.coords) {
            e = e.coords;
        }
        section.setItems([
            { properties: { title: 'LOCATION:\n' + e.latitude + ', ' + e.longitude, color: '#00BFFF' } },
            { properties: { title: 'LOCATION ACCURACY:\n' + e.accuracy, color: '#00BFFF' } },
            { properties: { title: 'ALTITUDE:\n' + e.altitude, color: '#00BFFF' } },
            { properties: { title: 'ALTITUDE ACCURACY:\n' + e.altitudeAccuracy, color: '#00BFFF' } }
        ]);
    } else {
        section.setItems([
            { properties: { title: 'ERROR:\n' + e.error, color: 'red' } }
        ]);
    }
    listView.appendSection(section);
}

function getLocation () {
    Ti.Geolocation.addEventListener('location', push);
}

win.addEventListener('open', _ => {
    Ti.Geolocation.requestLocationPermissions(Ti.Geolocation.AUTHORIZATION_ALWAYS, e => {
        if (e.success) {
            getLocation();
        } else {
            alert('Could not obtain location permissions.');
        }
    });
});

win.add(listView);
win.open();
```
- Should not see `null` as `altitudeAccuracy` on Android 26+

[JIRA Ticket](https://jira.appcelerator.org/browse/TIMOB-20362)

### CODE REVIEW

File: android/modules/geolocation/src/java/ti/modules/titanium/geolocation/GeolocationModule.java [hunk 1]
```diff
@@ -786,14 +786,19 @@ private boolean shouldUseUpdate(Location newLocation)
...
+	@SuppressWarnings("NewApi")
 	private KrollDict buildLocationEvent(Location location, LocationProvider locationProvider)
 	{
 		KrollDict coordinates = new KrollDict();
 		coordinates.put(TiC.PROPERTY_LATITUDE, location.getLatitude());
 		coordinates.put(TiC.PROPERTY_LONGITUDE, location.getLongitude());
 		coordinates.put(TiC.PROPERTY_ALTITUDE, location.getAltitude());
 		coordinates.put(TiC.PROPERTY_ACCURACY, location.getAccuracy());
-		coordinates.put(TiC.PROPERTY_ALTITUDE_ACCURACY, null); // Not provided
+		if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
```
Comment (sgtcoolguy) [inline android/modules/geolocation/src/java/ti/modules/titanium/geolocation/GeolocationModule.java]: Personally I'd prefer to see the actual api version it is supported from. I'm not sure too many people are familiar with the "brand name" to api level mappings.

**Review by sgtcoolguy — CHANGES_REQUESTED** [review]
Please add to apidocs

File: android/modules/geolocation/src/java/ti/modules/titanium/geolocation/GeolocationModule.java (hunk 1 above)
Comment (garymathews) [reply android/modules/geolocation/src/java/ti/modules/titanium/geolocation/GeolocationModule.java]: I think the API version is less important here, it's nice to see what Android version supports altitude accuracy (i.e most people know Android Oreo is 8.0, where `26` would be less obvious).

**Review by sgtcoolguy — APPROVED** [review]

**Comment by ssaddique** [comment]
FR: Passed.

**Test Environment**
Studio Ver:     5.1.4.201909061933
SDK Ver:        8.3.0 (PR-11275)
OS Ver:         10.14.6
Appc NPM:       4.2.15
Appc CLI:       7.1.1
Node Ver:       10.16.3
NPM Ver:        6.11.3
Emulator:       ⇨ Pixel 3 (Android API 26-29)

**Pull request merged**